doc/source/examples.rst

```diff
@@ -140,6 +140,18 @@ You can run these tests from the nagios master or in the target host with
 `NRPE <https://en.wikipedia.org/wiki/Nagios#Nagios_Remote_Plugin_Executor>`_.
 
 
+Integration with KitchenCI
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+KitchenCI (aka Test Kitchen) can use testinfra via its :code:`shell` verifier. Just
+copy `conftest.py <conftest.py>`_ into your :code:`test` folder and set the verifier
+in your :code:`.kitchen.yml`::
+
+    verifier:
+      name: shell
+      command: testinfra -vvv --junit-xml "junit-${KITCHEN_INSTANCE}.xml" "test/integration/${KITCHEN_SUITE}"
```

Comment:
I tested this and could not determine an immediate performance impact. Following your suggestion I also “whitelisted” `ssh_identity_file` for the `hostspec` query so it actually ends up in `kw`. Work great, happy to have gotten rid of a redundant file!